exercises/practice/yacht/mix.exs

```diff
@@ -0,0 +1,27 @@
+defmodule Yacht.MixProject do
+  use Mix.Project
+
+  def project do
+    [
+      app: :yacht,
+      version: "0.1.0",
+      # elixir: "~> 1.12",
+      start_permanent: Mix.env() == :prod,
+      deps: deps()
+    ]
+  end
+
+  # Run "mix help compile.app" to learn about applications.
+  def application do
+    [
+      extra_applications: [:logger]
+    ]
+  end
+
+  # Run "mix help deps" to learn about dependencies.
+  defp deps do
+    [
+      {:jason, "~> 1.2"}
+    ]
+  end
```

Comment:
We can't use external dependencies in the test runner presently. See note down below about this

Reply:
No problem, I will update the PR without the dependency. I mostly did this as a fun exercise for myself so please don't worry about it not being documented explicitly (the contributing documentation is very helpful). I had a feeling it might not fit in the project as-is but I'm happy to make the changes so it does.